FILE README.md
RANGE {"path":"README.md","old_lines":[66,68],"new_lines":[66,68],"diff_block_lines":[5,9],"hunk_header":"@@ -66,3 +66,3 @@ npm run typecheck","substantive_changed_lines":1}
@@ -66,3 +66,3 @@ npm run typecheck
 ## License
 
-MIT
+AGPL-3.0
